osu.Game/Beatmaps/Drawables/Cards/BeatmapCard.cs

```diff
@@ -34,14 +35,19 @@ public class BeatmapCard : OsuClickableContainer
         private const float width = 408;
         private const float height = 100;
         private const float corner_radius = 10;
+        private const float icon_area_width = 30;
 
         private readonly APIBeatmapSet beatmapSet;
         private readonly Bindable<BeatmapSetFavouriteState> favouriteState;
 
+        [Cached]
+        private readonly BeatmapDownloadTracker downloadTracker;
```

Comment:
```suggestion
        private readonly BeatmapDownloadTracker downloadTracker;
```

I think this can be dropped now.